support/str/str.go

```diff
@@ -2,24 +2,24 @@ package str
 
 import (
 	"bytes"
-	"math/rand"
+	"crypto/rand"
 	"strconv"
 	"strings"
 	"unicode"
-
-	supporttime "github.com/goravel/framework/support/time"
 )
 
 func Random(length int) string {
```

Comment:
S: Could you please add a test case for `length == 0`? I forgot to add it before。

Reply:
> S: Could you please add a test case for `length == 0`? I forgot to add it before。

ok